src/resources/test_data_read_group.json

```diff
@@ -0,0 +1,170 @@
+[
```

Comment:
Should this and the other new json file be committed to the repo? Or were they just for testing?